components/exercises/chapter-4-oscar/AList/arrayToList.test.tsx

```diff
@@ -0,0 +1,57 @@
+import { arrayToList } from './arrayToList';
+
+describe('arrayToList', () => {
+  it('should convert an array of numbers into a linked list', () => {
+    const arr = [1, 2, 3];
+    const expectedList = {
+      value: 1,
+      rest: {
+        value: 2,
+        rest: {
+          value: 3,
+          rest: null,
+        },
+      },
+    };
+    expect(arrayToList(arr)).toEqual(expectedList);
+  });
+
+  it('should convert an array into a linked list', () => {
+    const arrString = ['a', 'b', 'c'];
```

Comment:
Let's do
```js
const arrString = ['a', 'b', 'c', 'd', 'e'];
```
to test for different input lengths (what if it only works for inputs with length 3?)